hugo-node/h0.104.3-n16.17.1/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:10-slim AS builder
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    --no-install-recommends
+
+RUN curl -L -o /tmp/node.tar.xz "https://nodejs.org/dist/v16.17.1/node-v16.17.1-linux-x64.tar.xz" \
+    && mkdir -p /usr/local/lib/nodejs \
+    && tar -xJf /tmp/node.tar.xz -C /usr/local/lib/nodejs \
+    && rm -f /tmp/node.tar.xz \
+    && ln -s /usr/local/lib/nodejs/node-v16.17.1-linux-x64/bin/node /usr/bin/node \
+    && ln -s /usr/local/lib/nodejs/node-v16.17.1-linux-x64/bin/npm /usr/bin/npm \
+    && ln -s /usr/local/lib/nodejs/node-v16.17.1-linux-x64/bin/npx /usr/bin/npx \
+    && npm install -g yarn@1.22.10 \
+    && ln -s /usr/local/lib/nodejs/node-v16.17.1-linux-x64/bin/yarn /usr/bin/yarn
+
+RUN curl -L -o /tmp/hugo.deb "https://github.com/gohugoio/hugo/releases/download/v0.104.3/hugo_0.104.3_linux-amd64.deb" \
```

Comment:
@mbarbero previously, we would have used https://github.com/gohugoio/hugo/releases/download/v0.104.3/hugo_0.104.3_Linux-64bit.deb

However, it longer exist. 

https://github.com/gohugoio/hugo/releases/download/v0.104.3/hugo_0.104.3_Linux-64bit.deb seems like the next reasonable URL to use. 

WDYT?

Reply:
you mean https://github.com/gohugoio/hugo/releases/download/v0.104.3/hugo_0.104.3_linux-amd64.deb?

Then it's fine to me

Reply:
Let me re-phrase, is there a different link I should be using here?
https://github.com/gohugoio/hugo/releases/tag/v0.104.3

Reply:
nop, it's all good.

Reply:
Awesome!